<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Information</class>
 <widget class="QMainWindow" name="Information">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>学生信息</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>611</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonAdd">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>0</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>添加</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonDel">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>160</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonOpen">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>260</y>
        <width>91</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>打开</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonSave">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>380</y>
        <width>91</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditSearch">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonVague">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>100</y>
        <width>115</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>模糊搜索</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonAdvanced">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>100</y>
        <width>115</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>高级搜索</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonFind">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>440</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>查找学生选课</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonSearch">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>40</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonRefresh">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>100</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditOpen">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>220</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditSave">
      <property name="geometry">
       <rect>
        <x>622</x>
        <y>320</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditDelete">
      <property name="geometry">
       <rect>
        <x>622</x>
        <y>120</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>129</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonChange">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>60</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>修改</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>课程信息</string>
     </attribute>
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>611</width>
        <height>521</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonRefresh2">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>70</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="actionquit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionquit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
